specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/examples/IntegrationRuntimeObjectMetadata_Get.json

```diff
@@ -39,390 +39,6 @@
             "id": 4,
             "name": "这是文件夹",
             "description": ""
```

Comment:
This example needs to be re-captured after Xiao's fix.

Reply:
It should be done in Xiao's PR.